_studio/mfx_lib/mctf_package/mctf/include/mctf_common.h

```diff
@@ -79,7 +79,7 @@ enum { MCTF_BITRATE_MULTIPLIER = 100000 };
 #include <cassert>
 #define CHROMABASE      80
 #define MAXCHROMA       100
-#define MCTFSTRENGTH    20
+#define MCTFSTRENGTH    3
```

Comment:
Why do we changing strength in this PR?

Reply:
This value is used only for MCTF in AVC pipeline, it does not affect VPP MCTF. Value 20 is just too high for Intra denoising, extra testing performed showed that balancing the spatial and temporal denoise strengths improve compression efficiency.